docs/contributing/contributing-releases/README.md

```diff
@@ -86,19 +86,16 @@ Let's say we have a bug in a release which needs to be patched for an already cr
 
 1. If breaking changes have been made to our Bicep fork: 
    
-   Update the file radius/.github/workflows/validate-bicep.yaml to use the release version (eg. v0.21) instead of edge for validating the biceps in the docs and samples repositories. Also modify the version from `env.REL_CHANNEL` to `<major>.<minor>` (eg. 0.21) for downloading the `rad-bicep-corerp`.
+   Update the file radius/.github/workflows/validate-bicep.yaml to use the release version (eg. v0.21) instead of edge for validating the biceps in the docs and samples repositories. Also modify the version from `env.REL_CHANNEL` to `<major>.<minor>` (eg. `0.21`) for downloading the `rad-bicep-corerp`.
 
-1. Push the commit to the remote and create a pull request targeting the release branch.
+1. Update the `versions.yaml` file with the new patch version (eg. replace `0.21.0` with `0.21.1` for the first patch, `0.21.1` with `0.21.2` for the second patch, etc)
+
+1. Push the commits to the remote and create a pull request targeting the release branch.
    ```bash
    git push origin <USERNAME>/<BRANCHNAME>
    ```
 
-1. After pull request is approved, merge into the release branch and tag!
-   ```bash
-   # replace v0.21.X with the version we want to patch (if we release 0.21.1 already, we would then release 0.21.2, etc.)
-   git tag v0.21.X
-   git push --tags
-   ```
+1. In the `main` branch, update the `versions.yaml` file with the patch version that was specified in the release branch (eg. `0.21.1`). This will kick off a new patch release. The release branch must be merged before this change is merged into main. 
```

Comment:
Inline to the above comment -->  I think here you have updated this but would be good to update this with a reasoning from your comment so we know - "...since the workflow in main branch builds the release based on the head of the release branch"